ams/server/src/main/java/com/netease/arctic/server/dashboard/PaimonTableDescriptor.java

```diff
@@ -40,7 +56,73 @@ public List<TableFormat> supportFormat() {
 
   @Override
   public ServerTableMeta getTableDetail(AmoroTable<?> amoroTable) {
-    throw new UnsupportedOperationException();
+    FileStoreTable table = getTable(amoroTable);
+    AbstractFileStore<?> store = (AbstractFileStore<?>) table.store();
+
+    ServerTableMeta serverTableMeta = new ServerTableMeta();
+    serverTableMeta.setTableIdentifier(amoroTable.id());
+    serverTableMeta.setTableType(amoroTable.format().name());
+
+    //schema
+    serverTableMeta.setSchema(
+        table.rowType().getFields().stream()
+            .map(s -> new AMSColumnInfo(s.name(), s.type().asSQLString(), !s.type().isNullable(), s.description()))
+            .collect(Collectors.toList())
+    );
+
+    //pk
```

Comment:
NIT: using `primary key` is more in line with English grammar.

Reply:
yeah